tests/wpt/web-platform-tests/html/rendering/non-replaced-elements/tables/table-width.html

```diff
@@ -0,0 +1,25 @@
+<link rel="match" href="table-width-ref.html">
```

Comment:
Thanks for the suggestion, that's a lot cleaner! Updated in the latest force push